app/views/events/_applicants_overview.html.erb

```diff
@@ -34,13 +34,14 @@
     </thead>
     <tbody>
     <%
-      @application_letters.sort_by! {|l| l.user.profile.send(params[:sort]) } if params[:sort]
+      @application_letters.sort_by! {|l| l.user.profile.send(params[:sort]) } if params[:sort] && params[:sort] != 'applicant_age_when_event_starts'
+      @application_letters.sort_by! {|l| l.send(params[:sort]) } if params[:sort] && params[:sort] == 'applicant_age_when_event_starts'
       @application_letters.reverse! if params[:order] == 'descending'
       @application_letters.select {|l| l.user.profile }.each do | application_letter | %>
```

Comment:
Though, the sorting needs to be done on client-side. So the only option is view, right?